### PR TITLE
Fix #26

### LINKED ISSUE #26: inconsitency adding lat labelling on the right (on graticules)...
#### Code Sample, a minimal, complete, and verifiable piece of code

```python
# Your code here
font=aggdraw.Font('yellow','/usr/share/fonts/truetype/dejavu/DejaVuSerif.ttf', opacity = 200, size=30)
areaid = 'eurol'
#areaid = 'germ'
lscn = scn.resample(areaid, radius_of_influence=10000)

lscn.save_dataset(composite, imgdir + '/Metop-A.jpg',
                          overlay={'coast_dir': '/satpy/coastlines/', 'color': (255, 255, 0), 'width': 1.0, 'resolution': 'i',
                          'grid': {'major_lonlat': (10, 10), 'minor_lonlat':(2, 2),'font': font,
                          'write_text': True,
                          'outline': (224, 224, 0), 'outline_opacity': 175,
                          'width': 1.0, 'minor_width':0.5, 'minor_outline_opacity': 175, 'minor_is_tick': False, 'lon_placement':'tb', 'lat_placement': 'lr'}}, fill_value=0)
```
#### Problem description
There seems to be some strange inconsitency adding lon/lat labeling in satpy.
If I try to place the labeling with
'lon_placement':'tb', 'lat_placement': 'lr'
and area = eurol
Pic1-Metop-A-1

the '20N' lable is missing on the lower right corner 

If I set e.g.
area = germ

the lat label '50N' is missing on the right. I can't get a labeling on the right (pic2)
I even tried to set ''lat_placement': 'lr' " or 'lat_placement': 'r'. But no labeling on the right.

Pic2-Metop-A
 
Another example, 40N, 50N and 60N missing, 'lat_placement': 'lr' 

Noaa_20-Orbit_7372-NightTime-20190422-M15-omerc_bb

#### Versions of Python, package at hand and relevant dependencies
satpy-0.14.1
pycoast-1.2.2
Python 3.6

### PR DESCRIPTION
**Pull request by mraspaud**
- [x] Closes #26  
 - [x] Tests added 
 - [x] Tests passed 
 - [x] Passes ``git diff origin/develop **/*py | flake8 --diff``

### CODE REVIEW

File: pycoast/cw_base.py [hunk 1]
```diff
@@ -253,7 +253,7 @@ def _add_grid(self, image, area_def,
 
         # lons along major lat lines (extended slightly to avoid missing the
         # end)
-        lin_lons = np.arange(lon_min, lon_max + Dlon / 5.0, Dlon / 10.0)
+        lin_lons = np.arange(lon_min, lon_max + Dlon / 5.0, Dlon / 100.0)
```
Comment (djhoese) [inline pycoast/cw_base.py]: Doesn't this mean that there will be a lot more lines in the final output image?

Could `linspace` be used instead to make sure that the last value is included? https://docs.scipy.org/doc/numpy/reference/generated/numpy.linspace.html

**Comment by peters77** [comment]
Tested with some areas. LGTM. :-)

**Comment by mraspaud** [comment]
@peters77 thanks for taking the time!

**Comment by mraspaud** [comment]
Does someone have an ubuntu system and can regenerate the images to comply with the new code ?

**Comment by djhoese** [comment]
"Yeah who has an ubuntu system...damn it."

I'll do it.

**Comment by mraspaud** [comment]
Yes, that's the idea

**Pull request merged**